### DISCUSSION TITLE
MQTT - Home Assistant - Set Electricity Price

### DISCUSSION THREAD

**Discussion by alexapps1** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#730] [Ideas]
In the same way home assistant via MQTT can be used to set ambient temperature, the ability the set the current 'Price per kWh' would be great for accurate cost tracking.

Thank you for all the hard work that's gone into this. It's brilliant 👏.

**Comment by visualapproach** [comment]
service: number.set_value
data:
  value: "{{states('sensor.electricity_price')}}"
target:
  entity_id: number.layzspa_electricity_price


You're welcome ☺️

**Comment by alexapps1** [comment]
Amazing. Thank you so much. Works perfectly of course 👌.
